packages/amplication-server/src/core/git/git.provider.service.ts

```diff
@@ -379,9 +383,16 @@ export class GitProviderService {
   async createGitOrganization(
     args: CreateGitOrganizationArgs
   ): Promise<GitOrganization> {
+    if (args.data.gitProvider !== EnumGitProvider.Github) {
```

Comment:
Is this on purpose not allowing BitBucket yet?

Reply:
We should refactor this service method since it is prone to mistake/misunderstanding. 
The reason is that we use a different flow for bitbucket (OAuth authorisation grant flow) from github (that is using a custom Github App flow)